utils/status_test_util.h

```diff
@@ -21,72 +21,12 @@ limitations under the License.
 #include "gtest/gtest.h"
 
 // Macros for testing the results of functions that return
```

Comment:
nit: Incomplete comment. 
The next comment can be removed too since EXPECT_OK is not defined in protobuf/stubs/status.h. https://github.com/search?q=repo%3Aprotocolbuffers%2Fprotobuf%20EXPECT_OK&type=code

Reply:
Done.